atlas/packages/components/src/components/VideoPlayer/VideoPlayer.style.tsx

```diff
@@ -2,32 +2,32 @@ import { css } from "@emotion/core";
 import { breakpoints } from "../../theme";
 
 export type VideoPlayerStyleProps = {
-  width?: string | number;
-  height?: string | number;
-  responsive?: boolean;
-  ratio?: string;
+	width?: string | number;
```

Comment:
Why these indent changes? I thought we are using 2 spaces? There's a couple more files affected by this